examples/js/loaders/GLTFLoader.js

```diff
@@ -535,6 +550,46 @@ THREE.GLTFLoader = ( function () {
 
 	};
 
+	/**
+	 * BasisU Texture Extension
+	 *
+	 * Specification: https://github.com/KhronosGroup/glTF/tree/master/extensions/2.0/Khronos/KHR_texture_basisu
+	 * (Link above TBD, currently in PR https://github.com/KhronosGroup/glTF/pull/1751)
+	 */
+	function GLTFTextureBasisU( parser ) {
+
+		this.parser = parser;
+		this.name = "KHR_texture_basisu";
```

Comment:
Could you add this to the `EXTENSIONS` enum, instead?

Reply:
Done